udata/core/user/search.py

```diff
@@ -48,6 +49,7 @@ class UserSearch(ModelSearchAdapter):
         'followers': Sort('metrics.followers'),
         'views': Sort('metrics.views'),
         'created': Sort('created'),
+        'roles': Sort('roles'),
```

Comment:
I think filtering is sufficient, sorting on `roles` does not seems relevant to me:
- a user can have many roles: which one is taken in account ?
- this is not a very variable value: there will be an unordered block of admins then others (and the opposite if the sort is reversed)